File: mis_conexiones/php/Mysql.php
<?php
$servername = "localhost";
$username = "acroxmen";
$password = "12345";
$dbname = "tienda";

$conn = new mysqli($servername, $username, $password, $dbname);

if ($conn->connect_error) {
    die("Conexión fallida: " . $conn->connect_error);
}

$sql = "SELECT * FROM tabla";
$result = $conn->query($sql);

if ($result->num_rows > 0) {
    while($row = $result->fetch_assoc()) {
        echo "id: " . $row["id"]. " - Nombre: " . $row["nombre"]. "<br>";
    }
} else {
    echo "0 resultados";
}
$conn->close();
?>

File: mis_conexiones/rubi/mysql.rb
require 'mysql2'

client = Mysql2::Client.new(
  host: 'localhost',
  username: 'acroxmen',
  password: '12345',
  database: 'mi_clase'
)

results = client.query('SELECT * FROM tabla')
results.each do |row|
  puts row
end
